examples/tv-casting-app/tv-casting-common/core/CastingPlayer.h

```diff
@@ -206,6 +223,11 @@ class CastingPlayer : public std::enable_shared_from_this<CastingPlayer>
     unsigned long long int mCommissioningWindowTimeoutSec = kCommissioningWindowTimeoutSec;
     ConnectCallback mOnCompleted                          = {};
 
+    /**
+     * @brief resets this CastingPlayer's state and calls mOnCompleted with the CHIP_ERROR.
```

Comment:
it also clears mOnCompleted ... not sure if this is obvious from the description. May want to expand this.

Reply:
Sounds good.